content/enterprise/admin/infrastructure/logging.mdx

```diff
@@ -334,4 +334,28 @@ recommend forwarding all Terraform Enterprise logs to a log aggregation system,
 filtering the audit logs based on the `[Audit Log]` string, and forwarding just
 the audit logs to the desired destination.
 
--> **Note:** If you are using a version of Terraform Enterprise prior to v202109-1, please refer to the [previous version of this page](/enterprise/admin/logging-old) with logspout information.
+## Log Rotation
+
+Log forwarding uses the `journald` Docker logging driver to send Terraform
+Enterprise logs to `systemd-journald`. This can cause increased disk utilization
+for the `/var/log/journal` path.
+
+To limit this disk utilization, configure the `SystemMaxFileSize` and
```

Comment:
Beautifully done!  Two thoughts:
- What do we mean by "reasonable values"? I wonder if we could say something more specific OR simplify to say "To limit disk use, set the `SystemMaxFileSize` and `SystemMaxFiles` settings within `etc/systemd/journald.conf`.
- I would move the explanation of what the example does to before the example. So I'd say "The following example tells..."   I think it's nice to tell folks what they're about to see before they see it.

Reply:
I like it! Thank you so much!